t/plugin/uri-blocker.t

```diff
@@ -332,3 +332,39 @@ GET /hello?name=;union%20select%20
 GET /hello?cc=2
 --- no_error_log
 [error]
+
+
+
+=== TEST 16: invalid rejected_msg length or type
+--- config
+location /t {
+    content_by_lua_block {
+        local data = {
+            {
+                input = [[{"plugins":{"uri-blocker":{"block_rules":["^a"],"rejected_msg":""}},"uri":"/hello"}]],
+                output = [[{"error_msg":"failed to check the configuration of plugin uri-blocker err: property \"rejected_msg\" validation failed: string too short, expected at least 1, got 0"}]]
+            },
+            {
+                input = [[{"plugins":{"uri-blocker":{"block_rules":["^a"],"rejected_msg":true}},"uri":"/hello"}]],
+                output = [[{"error_msg":"failed to check the configuration of plugin uri-blocker err: property \"rejected_msg\" validation failed: wrong type: expected string, got boolean"}]]
```

Comment:
Could you change them to tables instead of long strings?

Reply:
seem `uri-blocker` can't use the key ~

Reply:
What key?

Reply:
table property ~

![image](https://user-images.githubusercontent.com/75366457/130177443-240ffe51-e683-4850-a42f-2f510d74c66e.png)

Reply:
You can use `["uri-blocker"] = `

Reply:
Got~ Thanks~ :smiley: